src/apps/bridge/aanderaaController.cpp

```diff
@@ -252,13 +252,15 @@ static void runController(void *param) {
             log_buflen = snprintf(log_buf, SENSOR_LOG_BUF_SIZE,
                               "%s,"
```

Comment:
Mind adding a comment for "timemstamp" here? I forgot 😛

Reply:
Oh yeah, good call, totally, will do.